arcgis-ios-sdk-samples/Scenes/Create terrain surface from a local raster/CreateTerrainSurfaceFromLocalRasterViewController.swift

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import ArcGIS
+
+class CreateTerrainSurfaceFromLocalRasterViewController: UIViewController {
+    @IBOutlet weak var sceneView: AGSSceneView! {
+        didSet {
+            //initialize scene
+            let scene = AGSScene(basemapType: .imageryWithLabels)
+            sceneView.scene = scene
+            let camera = AGSCamera(latitude: 36.525, longitude: -121.80, altitude: 300.0, heading: 180, pitch: 80, roll: 0)
+            sceneView.setViewpointCamera(camera)
+            
+            //add a raster to the surface
```

Comment:
Here we are first creating a raster elevation source (which is different from a raster :)) and then adding that to the surface. Could you change the comment accordingly?